docs/src/standard_results/structure_depth_surfaces.md

```diff
@@ -0,0 +1,48 @@
+# Structure depth surfaces
+
+This exports the modelled structural depth surfaces from within RMS.
+These surfaces typically represent the final surface set generated during a structural
+modeling workflow (after well conditioning), and frequently serve as the framework for
+constructing the grid.
+
+note::
+It is only possible to export **one single set** of depth surface predictions per model 
+workflow. They represent the structural prediction of the model in depth. 
+
+:::{table} Current
+:widths: auto
+:align: left
+
+| Field | Value |
+| --- | --- |
+| Version | NA |
+| Output | `share/results/maps/structure_depth_surfaces/surfacename.gri` |
+:::
+
+## Requirements
+
+- RMS
+- depth surfaces stored in a horizon folder within RMS
+
+The surfaces must be located within a horizon folder in RMS and be in domain `depth`.
+This export function will automatically export all non-empty horizons from the provided folder.
+
+
+## Usage
+
```

Comment:
Perhaps add here that you need 

`from fmu.dataio.export.rms import  export_structure_depth_surfaces`